src/components/material/MaterialDescription.tsx

```diff
@@ -65,6 +73,17 @@ const MaterialDescription: React.FC<MaterialDescriptionProps> = ({ work }) => {
             </p>
           )}
           <div className="material-description__links mt-32">
+            {!isFiction && dk5MainEntry && (
+              <HorizontalTermLine
```

Comment:
I wonder about the ordering of this element in relation to the order horizontal term lines.

This adds it as the first form line. I had expected expected it to be in relation to the `fictionNonfiction` one as it is relevant to non-fiction materials. If the current position has been cleared with the business then OK.